dashboard/app/controllers/api/v1/pd/applications_controller.rb

```diff
@@ -71,11 +71,22 @@ def quick_view
   # GET /api/v1/pd/applications/cohort_view?role=:role&regional_partner_filter=:regional_partner_name
   def cohort_view
     applications = get_applications_by_role(params[:role].to_sym).where(status: ['accepted', 'withdrawn'])
+    cohort_capacity = nil
```

Comment:
Consider initializing this to 0 instead of nil

Reply:
To prevent errors later on when this value is used, or for some other reason?